jvcl/run/JvWndProcHook.pas

```diff
@@ -39,13 +34,12 @@ interface
   {$IFDEF UNITVERSIONING}
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
-  Windows, Messages, SysUtils, Controls, Forms, Classes,
-  JvComponentBase;
+  Windows, Messages, SysUtils, Controls, Forms, Classes, JvComponentBase;
 
 type
-  TJvControlHook = function(var Msg: TMessage): Boolean of object;
-  TJvHookMessageEvent = procedure(Sender: TObject; var Msg: TMessage;
-    var Handled: Boolean) of object;
+  TJvControlHook       = function( var Msg: TMessage): Boolean of object;
```

Comment:
Please respect the style guide, no "equal alignment" and no space after an opening parenthesis